src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
```

Comment:
don't push the comments
```suggestion
```

Reply:
when I test it gives: Puppeteer encountered an error while running scenario "Link with data-qa_hover"; TimeoutError: waiting for selector "[data-qa="hover"]" failed: timeout 30000ms exceeded
what is wrong with data-qa="hover"?